lib/src/excel.dart

```diff
@@ -126,6 +137,32 @@ class ExcelDart {
     }));
   }
 
+  /// Highlight rows that are underpacing.
```

Comment:
Shall I change the relative notation used in this [function](https://github.com/googleinterns/dv360-excel-plugin/pull/53/commits/847e079cc3921677bce9ae2c909ac2f2e8fbb5d8#diff-5a9f11b722aad015b548cb227e5be190R118-R138) to match the regular notation instead? So instead of referencing RC[-2], we can use $M2 to reference the budget column instead. 

Both depends on hard coding though :<

Reply:
i think absolute > relative. But cant we calculate which column it is since we are specifying the schema? why hardcode?

Reply:
good point, good point, good point +1000.